icons/circle/48/telegram.svg

```diff
@@ -1,33 +1,15 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
+<svg version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you please add `width="48" height="48"` attributes to the `svg` declaration? That should fix the web preview.